### ISSUE TITLE
Update game server versions through confconsole

### ISSUE THREAD

**Issue by dragonfly8677** [turnkeylinux/tracker#1990]
Currently there's no way to update the server version of turnkey-gameserver through the ui. Instead you have to find the appropriate command, adjust it for your specific game server and run it in the appropriate folder. This would be better placed in the ui, as updating a game server version is fairly commonplace.

I propose adding a new option in the "Game Server" menu of the `confconsole`. The current options are:
 
![{DECE5A5F-EDDA-4801-A71F-20A64205AA71}](https://github.com/user-attachments/assets/f05a2678-e331-4b9c-bbf7-6289d022f7e7)

And this is how I imagine it:

- Select game
- Update server
- Update game list

The "Update server" is the new option, and the "Update game list" is the one that already exists, just renamed.

**Comment by dragonfly8677** [comment]
After using this more I've found a better way to update the server than the one that was implemented in my previous PR. I suggest we keep this ticket open until I make a new MR that uses these new techniques.

In my previous PR I used the `auto_install.sh` script to reinstall the existing gameserver. What I've since found is the following page: https://docs.linuxgsm.com/commands/update. This details a `./gameserver update` command, and similarly a `./gameserver check-update` command. This would allow us to properly update the server instead of reinstalling it, and also only perform the update if an update is available.

Now obviously, it's never quite so easy. `./gameserver` needs to be replaced with the name of the actual game's server, e.g. `fctrserver` for Factorio, but my previous PR already has code for identifying the name of the installed server so that should be no issue.

I therefore envision the following flow:

- The user selects `Update game` in confconsole
- Identify the installed server (the code up until this point already exists)
- Run `./gameserver check-update`
  - If no update is available, end
- Run `./gameserver update`

I'll make a PR for this asap.
